dataset/dictionary/locations.json5

```
@@ -1,11 +1,36 @@
 [
   {
-    en: "",
-    ja: "",
-    zhCN: "",
-    pronunciationJa: "",
-    notes: "",
-    notesZh: "",
-    tags: [ "location" ],
+    en: "Jarilo-VI",
```

I think Ⅵ (Roman figure) is better than V + I.
I will convert it to 6 when I generate the slug, so don't worry.

```suggestion
    en: "Jarilo-Ⅵ",
```